src/Common/src/System/LocalAppContext.cs

```diff
@@ -40,6 +38,17 @@ private static bool GetCachedSwitchValueInternal(string switchName, ref int swit
             return isSwitchEnabled;
         }
 
+        private static void InitDisableCaching()
+        {
+            bool isEnabled;
+            if (AppContext.TryGetSwitch(@"TestSwitch.LocalAppContext.DisableCaching", out isEnabled))
```

Comment:
But if we remove the setter then we would have to have either a private field to cache the obtained value or always call `AppContext.TryGetSwitch` to get the `DisableCaching` switch value. Do we want this behavior? I say we should cache the value in orther to avoid calling `TryGetSwitch` every time.